udf/round_timestamp_to_minute.sql

```diff
@@ -0,0 +1,24 @@
+/*
+
+Floor a timestamp object to the given minute interval.
+
+ */
+CREATE TEMP FUNCTION udf_round_timestamp_to_minute(timestamp_expression TIMESTAMP, minute INT64) AS (
+  TIMESTAMP_SECONDS(
+    CAST((FLOOR(UNIX_SECONDS(timestamp_expression) / (minute * 60)) * minute * 60) AS INT64)
```

Comment:
Can you add a test case for `udf_round_timestamp_to_minute(TIMESTAMP '2019-05-03 00:34:59.999999', 5)` to validate whether `UNIX_SECONDS` is potentially rounding up?